tensorflow/lite/micro/kernels/cmsis_nn/add.cc

```diff
@@ -103,19 +103,20 @@ TfLiteStatus CalculateOpData(TfLiteContext* context, TfLiteAddParams* params,
   return kTfLiteOk;
 }
 
-void UpdateOpParams(tflite::ArithmeticParams& op_params, const OpData* data) {
-  op_params.left_shift = data->left_shift;
-  op_params.input1_offset = data->input1_offset;
-  op_params.input1_multiplier = data->input1_multiplier;
-  op_params.input1_shift = data->input1_shift;
-  op_params.input2_offset = data->input2_offset;
-  op_params.input2_multiplier = data->input2_multiplier;
-  op_params.input2_shift = data->input2_shift;
-  op_params.output_offset = data->output_offset;
-  op_params.output_multiplier = data->output_multiplier;
-  op_params.output_shift = data->output_shift;
+void UpdateOpParams(tflite::ArithmeticParams* const op_params,
+                    const OpData* data) {
```

Comment:
is there a reason to change from reference to pointer in this PR?

Reply:
Reason was to fix lint errors.
https://github.com/tensorflow/tflite-micro/blob/main/CONTRIBUTING.md#before-submitting-your-pr